docs/data/date-pickers/getting-started/getting-started.md

```diff
@@ -76,20 +74,164 @@ The bundling strategy was taken care of by a Create React App, and no locale was
 :::
 
+### Other peer dependencies
+
+The Date and Time Pickers package has a peer dependency on `@mui/material`.
+If you are not already using it in your project, you can install it with:
+
+```sh
+// with npm
+npm install @mui/material @emotion/react @emotion/styled
+
+// with yarn
+yarn add @mui/material @emotion/react @emotion/styled
+```
+
+<!-- #react-peer-version -->
+
+Please note that [react](https://www.npmjs.com/package/react) >= 17.0.2 and [react-dom](https://www.npmjs.com/package/react-dom) >= 17.0.2 are peer dependencies.
+
+MUI is using [emotion](https://emotion.sh/docs/introduction) as a styling engine by default. If you want to use [`styled-components`](https://styled-components.com/) instead, run:
+
+```sh
+// with npm
+npm install @mui/material @mui/styled-engine-sc styled-components
+
+// with yarn
+yarn add @mui/material @mui/styled-engine-sc styled-components
+```
+
 ## Code setup
 
-After installation, you have to pass your chosen date library's adapter to `LocalizationProvider`.
-The supported adapters—as well as `LocalizationProvider`—are exported from both the `@mui/x-date-pickers` and `@mui/x-date-pickers-pro` packages.
+Before trying to render any component, you have to make sure that there is a `LocalizationProvider` upper in the React tree.
+This component receives your chosen date library's adapter and make it accessible to all the Date and Time Pickers component with a React context.
 
-All the pickers rendered inside this provider will have access to the adapter through a React context.
-For this reason, we recommend that you wrap your entire application with a `LocalizationProvider` so you can use the Date and Time Pickers everywhere.
+The most simple way to use `LocalizationProvider` is to wrap your entire application it.
+You will then be able to use the Date and Time Pickers everywhere.
 
 {{"component": "modules/components/PickersRenderingInstructions.js"}}
 
+:::success
+
+- All the adapters are exported by both `@mui/x-date-pickers` and `@mui/x-date-pickers-pro`:
+
+  ```tsx
+  import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+  import { AdapterDateFns } from '@mui/x-date-pickers-pro/`AdapterDayjs`';
+  ```
+
+- `LocalizationProvider` is exported by both `@mui/x-date-pickers` and `@mui/x-date-pickers-pro`:
+
+  ```tsx
+  import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+  import { LocalizationProvider } from '@mui/x-date-pickers';
+  import { LocalizationProvider } from '@mui/x-date-pickers-pro/LocalizationProvider';
+  import { LocalizationProvider } from '@mui/x-date-pickers-pro/';
+  ```
+
+:::
+
 :::info
 If you need to use the Date and Time Pickers with a custom locale, have a look at the [Localized dates](/x/react-date-pickers/adapters-locale/) page.
 :::
 
+## Commonly used components
+
+Most applications only need a few of the components exposed by the Date and Time packages:
+
+{{"demo": "CommonlyUsedComponents.js"}}
```

Comment:
Amazing demo for introducing and playing with components 🎉

But it's the first time reader encounter a demo, and the code is complicated to read.

Reply:
Yeah, this demo is interesting to help people find quickly their component, not to help them copy paste code...
We could split into 4 demos (this page has a ton of very basic demo anyway)

But my idea was that it was the "Controlled value" demo which should be the entry point for people to copy / paste, since it's closer from a real use case scenario.
I do agree that it's not very accessible right now though.